rq_dashboard/web.py

```diff
@@ -81,10 +114,10 @@ def setup_rq_connection():
         current_app.redis_conn = sentinel.master_for(redis_master)
     else:
         current_app.redis_conn = Redis(
-            host=current_app.config.get('RQ_DASHBOARD_REDIS_HOST'),
-            port=current_app.config.get('RQ_DASHBOARD_REDIS_PORT'),
+            host=current_app.config.get('RQ_DASHBOARD_REDIS_HOST', 'localhost'),
```

Comment:
`app.config` should be responsible for setting default values, not the rq connection code

Reply:
It came from upstream.

Reply:
Ugh...at least you & I know better ;) We'll keep it as is in that case